vaccine_feed_ingest/stages/enrichment.py

```diff
@@ -23,12 +24,25 @@ def enrich_locations(
     input_dir: pathlib.Path,
     output_dir: pathlib.Path,
     api_cache: Optional[diskcache.Cache] = None,
+    enrich_apis: Optional[Collection[str]] = None,
+    placekey_apikey: Optional[str] = None,
 ) -> bool:
     """Enrich locations in normalized input_dir and write them to output_dir"""
     enriched_locations = []
 
+    if enrich_apis is None:
+        enrich_apis = set()
+
+    placekey_api = None
+    if "placekey" in enrich_apis:
+        if api_cache is not None and placekey_apikey:
+            placekey_api = PlacekeyAPI(api_cache, placekey_apikey)
+        else:
+            logger.info("Skipping placekey because placekey api is not configured")
```

Comment:
nit: should this be `logger.warning` (or higher?)

Reply:
Yep, I upgraded to error because this means you requested that the placekey api be used, but we couldn't call it.